### ISSUE TITLE
`bikeshed source` nags about missing `bigblocks.bsfont` file

### ISSUE THREAD

**Issue by bramus** [speced/bikeshed#2484]
I’ve got python 3.10 installed using Brew. Running `bikeshed source` spits out this:

```bash
$ bikeshed source
FATAL ERROR: Couldn't find font file “/opt/homebrew/lib/python3.10/site-packages/bikeshed/bigblocks.bsfont”:
[Errno 2] No such file or directory: '/opt/homebrew/lib/python3.10/site-packages/bikeshed/bigblocks.bsfont'
 ✘  Did not generate, due to fatal errors
```

The file `bigblocks.bsfont` indeed seems to be missing:

```bash
$ ls -Al /opt/homebrew/lib/python3.10/site-packages/bikeshed
total 1128
-rw-r--r--   1 bramus  admin   9486 Dec 12 11:31 InputSource.py
-rw-r--r--   1 bramus  admin  32639 Dec 12 11:31 Spec.py
-rw-r--r--   1 bramus  admin   2385 Dec 12 11:31 __init__.py
-rw-r--r--   1 bramus  admin     30 Dec 12 11:31 __main__.py
drwxr-xr-x  37 bramus  admin   1184 Dec 12 11:31 __pycache__
-rw-r--r--   1 bramus  admin  18106 Dec 12 11:31 biblio.py
drwxr-xr-x   3 bramus  admin     96 Dec 12 11:31 boilerplate
-rw-r--r--   1 bramus  admin  46056 Dec 12 11:31 boilerplate.py
drwxr-xr-x   8 bramus  admin    256 Dec 12 11:31 caniuse
-rw-r--r--   1 bramus  admin  26708 Dec 12 11:31 cli.py
-rw-r--r--   1 bramus  admin   3886 Dec 12 11:31 conditional.py
drwxr-xr-x   9 bramus  admin    288 Dec 12 11:31 config
-rw-r--r--   1 bramus  admin    874 Dec 12 11:31 constants.py
-rw-r--r--   1 bramus  admin  45235 Dec 12 11:31 datablocks.py
-rw-r--r--   1 bramus  admin   9291 Dec 12 11:31 dfnpanels.py
drwxr-xr-x   5 bramus  admin    160 Dec 12 11:31 dfns
-rw-r--r--   1 bramus  admin    232 Dec 12 11:31 extensions.py
-rw-r--r--   1 bramus  admin   2845 Dec 12 11:31 fingerprinting.py
-rw-r--r--   1 bramus  admin   8519 Dec 12 11:31 fonts.py
-rw-r--r--   1 bramus  admin    368 Dec 12 11:31 func.py
drwxr-xr-x   7 bramus  admin    224 Dec 12 11:31 h
-rw-r--r--   1 bramus  admin   5028 Dec 12 11:31 headings.py
-rw-r--r--   1 bramus  admin  27602 Dec 12 11:31 highlight.py
-rw-r--r--   1 bramus  admin  23978 Dec 12 11:31 idl.py
-rw-r--r--   1 bramus  admin   7495 Dec 12 11:31 includes.py
drwxr-xr-x   4 bramus  admin    128 Dec 12 11:31 inlineTags
-rw-r--r--   1 bramus  admin  10841 Dec 12 11:31 issuelist.py
-rw-r--r--   1 bramus  admin    185 Dec 12 11:31 language.py
-rw-r--r--   1 bramus  admin   1582 Dec 12 11:31 lexers.py
-rw-r--r--   1 bramus  admin    397 Dec 12 11:31 line.py
drwxr-xr-x  10 bramus  admin    320 Dec 12 11:31 lint
drwxr-xr-x   5 bramus  admin    160 Dec 12 11:31 markdown
drwxr-xr-x   8 bramus  admin    256 Dec 12 11:31 mdn
-rw-r--r--   1 bramus  admin   6871 Dec 12 11:31 messages.py
-rw-r--r--   1 bramus  admin  55637 Dec 12 11:31 metadata.py
-rw-r--r--   1 bramus  admin   2689 Dec 12 11:31 publish.py
-rw-r--r--   1 bramus  admin    973 Dec 12 11:31 pyg.py
-rw-r--r--   1 bramus  admin  47048 Dec 12 11:31 railroaddiagrams.py
-rw-r--r--   1 bramus  admin   8540 Dec 12 11:31 railroadparser.py
drwxr-xr-x  10 bramus  admin    320 Dec 12 11:31 refs
-rw-r--r--   1 bramus  admin   1292 Dec 12 11:31 repository.py
-rw-r--r--   1 bramus  admin   6739 Dec 12 11:31 retrieve.py
drwxr-xr-x   5 bramus  admin    160 Dec 12 11:31 shorthands
drwxr-xr-x  23 bramus  admin    736 Dec 12 11:31 spec-data
drwxr-xr-x   5 bramus  admin    160 Dec 12 11:31 stringEnum
-rw-r--r--   1 bramus  admin   1921 Dec 12 11:31 t.py
-rw-r--r--   1 bramus  admin   5183 Dec 12 11:31 test.py
-rw-r--r--   1 bramus  admin    527 Dec 12 11:31 testsuite.py
drwxr-xr-x   5 bramus  admin    160 Dec 12 11:31 translate
-rw-r--r--   1 bramus  admin  69763 Dec 12 11:31 unsortedJunk.py
drwxr-xr-x  16 bramus  admin    512 Dec 12 11:31 update
drwxr-xr-x   6 bramus  admin    192 Dec 12 11:31 wpt
```

**Comment by tabatkins** [comment]
Ah indeed, I wasn't including it in the build files. Surprised you're the first to complain about it; I guess the other people likely to use `bikeshed source` are probably running on Bikeshed trunk.

I've fixed the build, but am trying to bundle another change in before I cut a release; unless this takes too long, expect a fix in a few hours.

**Comment by tabatkins** [comment]
Should be fixed in 3.11.13.
